flasc/energy_ratio/energy_ratio.py

```diff
@@ -186,6 +193,10 @@ def _compute_energy_ratio_bootstrap(er_in,
         ws_min (float): The minimum wind speed to use.
         ws_max (float): The maximum wind speed to use.
         bin_cols_in (list[str]): A list of column names to use for the wind speed and wind direction bins.
+        weight_by (str): How to weight the energy ratio, options are 'min', or 'sum'.  'min' means
+            the minimum count across the dataframes is used to weight the energy ratio.  'mean' means the mean
+            count across the dataframes is used to weight the energy ratio.  'sum' means the sum of the counts
```

Comment:
I think this line about 'mean' means the mean count across... should be removed. It doesn't seem like the 'mean' option is implemented anywhere.